### DISCUSSION TITLE
blood cell:plasma partition coefficient

### DISCUSSION THREAD

**Discussion by rreissexp** [Open-Systems-Pharmacology/Forum#1501] [Questions & Problems]
I have a chemical with a logP=2.95 and I'm assuming no plasma protein binding.  The partition coefficient (blood cells/plasma) is 15.89 using the Schmitt method.  That seems unrealistically high.  Is this right or am I misinterpreting something.

**Comment by StephanSchaller** [comment]
@rreissexp , a logP of 3 is quite high and combined with no protein binding a high partitioning is expected. If you reduce fu to 10% (a common fu for such a lipophilicity value), partitioning will also reduce by about the same factor to 1.5 (maybe a value you are more likely to expect).
Where did you obtain your fu value?

**Comment by rreissexp** [comment]
Thanks.  We are still measuring fu; had stability issues the first run.  This is very helpful.
